q2cli/tests/test_core.py

```diff
@@ -10,23 +10,27 @@
 import shutil
 import tempfile
 import unittest
+import configparser
 
 from click.testing import CliRunner
 from qiime2 import Artifact
 from qiime2.core.testing.type import IntSequence1
 from qiime2.core.testing.util import get_dummy_plugin
 
 import q2cli
+import q2cli.util
 import q2cli.builtin.info
 import q2cli.builtin.tools
 from q2cli.commands import RootCommand
+from q2cli.core.config import CONFIG
```

Comment:
This can be tricky to unit-test since it's a module-level singleton. (your tests will end up side-effecting each other indirectly through the same shared object).

I would recommend constructing a new `CLIConfig` object for each test, or if you must test the module level variable, you could use [`importlib.reload`](https://docs.python.org/3/library/importlib.html#importlib.reload), but that's much more annoying to deal with.